experiments/examples/yaml/spider9.yaml

```diff
@@ -145,4 +145,15 @@ body:
                     blue: 0.72
                   orientation : 0
 brain:
-  type: bo-cpg
+  type: cppn-cpg
```

Comment:
this brain should not work, it has no learning or weights configured. What happened?